### PR TITLE
Port OAM tiler code base to Raster Foundry

### PR DESCRIPTION
**Pull request by hectcastro**
This is largely a copy and paste job, with a bunch of stray Bash script and dummy data removals. The plan is to do an initial port that successfully compiles into an assembly JAR and passes the test suite. The second pass will actually attempt to get things working and fill out the README in the process.

### CODE REVIEW

File: mosaic/src/test/scala/com/azavea/rasterfoundry/TilerSpec.scala [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+import geotrellis.spark.ingest._
+import geotrellis.spark.utils._
+
+import org.apache.spark._
+import spire.syntax.cfor._
+
+import org.scalatest._
+import geotrellis.testkit._
+
+// object TestSparkContext {
```
Comment (lliss) [inline mosaic/src/test/scala/com/azavea/rasterfoundry/TilerSpec.scala]: Why is this all commented out? Can we drop this file?

Comment (hectcastro) [inline mosaic/src/test/scala/com/azavea/rasterfoundry/TilerSpec.scala]: Maybe @echeipesh can provide some insight here.

**Comment by kdeloach** [comment]
Can we populate the boilerplate placeholders in the license?

**Comment by kdeloach** [comment]
+1

File: mosaic/src/test/scala/com/azavea/rasterfoundry/TilerSpec.scala (hunk 1 above)
Comment (echeipesh) [inline mosaic/src/test/scala/com/azavea/rasterfoundry/TilerSpec.scala]: My philosophy is: commented code does not exist. This bit doesn't even type check.

File: chunk/chunk.py [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+    (min_col, max_col) = (image_source.tile_bounds[0],
+                          image_source.tile_bounds[2])
+    (min_row, max_row) = (image_source.tile_bounds[1],
+                          image_source.tile_bounds[3])
+
+    for tile_col in range(min_col, min(max_col + 1, 2 ** zoom)):
+        for tile_row in range(min_row, min(max_row + 1, 2 ** zoom)):
+            tile_bounds = mercantile.bounds(tile_col, tile_row, zoom)
+            (wm_left, wm_bottom, wm_right, wm_top) = \
+                warp.transform_bounds("EPSG:4326",
```
Comment (echeipesh) [inline chunk/chunk.py]: looks like this may be a mistake that source is hard coded

**Pull request merged**